templates/CodeAnalysis.yml

```diff
@@ -55,16 +63,16 @@ stages:
       displayName: Build
       inputs:
         command: build
-        projects: ${{ parameters.solution }}
+        projects: ${{ parameters.solutions }}
         arguments: '--no-restore -c Release'
 
     - task: DotNetCoreCLI@2
       displayName: Run tests with Coverage
       timeoutInMinutes: 20
       inputs:
         command: test
-        projects: ${{ parameters.testProjects}}
-        arguments: '--no-restore --no-build -c Release -f net7.0 --filter Category!=WindowsOnly --collect:"XPlat Code Coverage" --settings coverlet.runsettings'
+        projects: ${{ parameters.testProjects }}
+        arguments: '--no-restore --no-build -c Release -f net6.0 --collect:"XPlat Code Coverage" --settings coverlet.runsettings'
 
```

Comment:
With some of the net8.0 features like keyed dependency injection, should we run the test coverage on net8.0?

Reply:
done